includes/Connector.class.php

```diff
@@ -358,4 +362,32 @@ protected function throwRequestException($message) {
 
 		throw $requestException;
 	}
+
+	/**
+	 * Checks the cURL request for errors and throws exceptions appropriately
+	 *
+	 * @param $request
+	 * @param $response string The response from the request
+	 * @throws RequestException
+	 * @throws ClientException
+	 * @throws ServerException
+	 * @throws TimeoutException
+	 */
+	protected function checkForRequestErrors($request, $response) {
+		// if curl timed out
+		if (curl_errno($request) && (string)curl_errno($request) === '28') {

```

Comment:
CURLE_OPERATION_TIMEDOUT
Let's use the curl constant rather than the number. Also maybe a switch inside the first if would make it clearer. Also the comment needs to reflect the block of code.

Reply:
@cristiangrama Updated!